src/client/index.js

```diff
@@ -45,10 +37,12 @@ export default function statinamic({
     <div id="statinamic-container">
       <MetadataProvider metadata={ metadata }>
         <Provider store={ store }>
-          <Router history={ history } routes={ routes } />
+          <div>
```

Comment:
@MoOx 
How would you avoid extra div ? 

devtools component needs to be inside Provider
and Provider only accepts a child